### DISCUSSION TITLE
problem on Catalina

### DISCUSSION THREAD

**Discussion by pierluigi-rinaldi** [iraf-community/iraf#149] [Q&A] [answered]
Hi everyone.

I am not able to install iraf on my Catalina.
I followed this guide: https://yucelkilic.com/irafv2_16_1_snapshot_installation_on_macos_mojave_10_14_6/?fbclid=IwAR1WNNm2GTTD29ik0TbemKgUU3FOVJejU3Qz3cacZ_aNoj2PN7TQSiDVRoM , but it doesn't work at all. For instance, I am not able to use mkiraf. When I try to do >which mkiraf it returns me 'mkiraf is not found', I don't understand why.

Any solution?

**Comment by olebole** [comment]
You don't need `mkiraf` anymore.
Did the compilation went through without errors?
Did you run the tests? Did they pass?

**Comment by pierluigi-rinaldi** [comment]
Mh, ok about 'mkiraf'. It's strange, that guide says to do it.
Anyway, I did ./runtest, and this is the result:
```
ecl.e: README.md 
ecl.e: files.md ......
ecl.e: images.imcoords.md ...........
ecl.e: images.imfilter.md .x............
ecl.e: images.imfit.md ...
ecl.e: images.imgeom.md ........
ecl.e: images.immatch.md ..
ecl.e: lists.md .......
ecl.e: noao.astutil.md ........
ecl.e: noao.digiphot.photcal.md .
ecl.e: numerical-recipes.md ..........
ecl.e: os.md ...
ecl.e: programming.md ...........
ecl.e: sys.vops.md .
ecl.e: test-syntax.md ...xs.
ecl.e: testproc.md .........................
ecl.e: utilities.nttools.md ..............
Test summary:  127 passed
   1 skipped
   2 xfailed
```

Is it ok? What am I supposed to do now?
Thanks for your time, and have a nice Sunday.

**Comment by olebole** [comment, marked as answer]
That is OK. Now you can run `ecl`, if it is in your path.
General recommendation: just **don't** start to work with IRAF. It is basically dead. It has no future at all. The reason to keep it is mainly to help those, who still depend on it in their workflow. But it is really highly recommended to use [Astropy](https://astropy.org) and to help there improving it where it still does not fit the needs.
For Astropy, there are lots of people which are able to help, while on IRAF you are basically alone.
